utils/check_repo.py

```diff
@@ -126,6 +126,8 @@
     "VisualBertForVisualReasoning",
     "VisualBertForQuestionAnswering",
     "VisualBertForMultipleChoice",
+    "TFWav2Vec2ForCTC",
+    "TFWav2Vec2Model",
```

Comment:
Agreed with `TFWav2Vec2ForCTC` being in that list but `TFWav2Vec2Model` should have be in the TFAutoModel class.

Reply:
I think that I fixed this. I commented and tagged you in the location of the change. Let me know if I missed something.